card/spec/models/card_spec.rb

```diff
@@ -129,8 +129,8 @@ def assert_stable card1
 
 
   it 'update_should_create_subcards' do
-    banana = Card.create! :name=>'Banana'
-    Card.update banana.id, :subcards=>{ "+peel" => { :content => "yellow" }}
+    banana = Card.create! name: 'Banana'
+    Card.update banana.id, subcards: { "+peel" => { content: "yellow" }}
```

Comment:
Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.